### PR TITLE
Landing page revision

### PR DESCRIPTION
**Pull request by DominikaK**

### CODE REVIEW

**Review by dew326 — APPROVED** [review]

File: docs/guide/page.md [hunk 1]
```diff
@@ -0,0 +1,68 @@
+# Pages
+
+eZ Commerce adds the following blocks for the Page Builder:
```
Comment (dabrt) [inline docs/guide/page.md]:
```suggestion
eZ Commerce adds the following blocks to the Page Builder:
```

File: docs/guide/page.md [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+
+- Last viewed items - list of products recently viewed by the current user
+- Bestsellers - list of bestsellers for a given product category
+- Product slider - slider for products grouped with tabs
+
+![Page](img/page_1.png)
```
Comment (dabrt) [inline docs/guide/page.md:9]:
```suggestion
![Page](img/landing_page_1.png)  
```
Also, last time I spoke to Magda, she insisted that all images should have a caption.

File: docs/guide/page.md [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+```
+
+After each call, the cache for the last viewed product slider is purged.
+By default, the view is rendered by an ESI block and cached per user.
+When the customer visits a new product, the list is purged and regenerated the next time it is displayed. 
+
+Products are stored in the session.
+
+#### Displaying products
+
```
Comment (dabrt) [inline docs/guide/page.md:40]:
```suggestion

To display a list of recently viewed products, place the following code in a template:

```

File: docs/guide/page.md [hunk 4]
```diff
@@ -0,0 +1,68 @@
...
+``` html+twig
+{{ render_esi(
+    controller(
+        'Silversolutions\\Bundle\\EshopBundle\\Controller\\EzFlowController::showLastViewedProductsAction',
+        { }
+    )) }}
+```
+
+The controller is able to render a different template if required (parameter template).
```
Comment (dabrt) [inline docs/guide/page.md]:
```suggestion
The controller is able to render a different template if required (a parameter template).
```

File: docs/guide/page.md [hunk 5]
```diff
@@ -0,0 +1,68 @@
...
+The caching strategy can be defined in the config file. The strategy is dynamic, so use `vary: cookie`.
+
+``` yaml
+silver_eshop.default.http_cache:
+    last_viewed_products:
+        max_age: 36000
+        vary: cookie
+```
+
+You can configure the maximum number of last viewed products in configuration
```
Comment (dabrt) [inline docs/guide/page.md]:
```suggestion
You can configure the maximum number of last viewed products by selecting
```

File: docs/guide/page.md [hunk 6]
```diff
@@ -0,0 +1,68 @@
...
+in **eCommerce** -> **Configuration Settings** -> **Catalog** -> **Last viewed product limit**.
```
Comment (dabrt) [inline docs/guide/page.md]:
```suggestion
**eCommerce** -> **Configuration Settings** -> **Catalog** -> **Last viewed product limit**.
```

**Review by dabrt — CHANGES_REQUESTED** [review]

File: docs/guide/page.md (hunk 2 above)
Comment (DominikaK) [reply docs/guide/page.md:9]: I changed the file name to correspond to current naming (see top of the diff).
Also, I think we must have an alt text for every image, but not a caption, when the surrounding text is clear.

Comment (dabrt) [reply docs/guide/page.md:9]: Now I see what you mean. We must then discuss the caption idea in the afternoon. However, the two images are different and the new one does not clearly show the three block types. Can we go back to it later and pick a clearer example?

Comment (DominikaK) [reply docs/guide/page.md:9]: Definitely, best after the next release.

**Review by dabrt — APPROVED** [review]

**Pull request merged**